hulk.js

```diff
@@ -156,20 +165,27 @@
         }
 
         var valueHTML = convertJSONToHTML(value, optionsCopy, key);
+        var hideToggle = getOptionOrDefault(options, "permissions", []);
+
         valueHTML.addClass('hulk-map-value-container');
         if (valueHTML.children('.hulk-map-pair, .hulk-array-element').length > 0) {
             var button = $(document.createElement('button'));
             button.addClass('hulk-collapse-item');
             if (depth !== 0) {
                 button.text("Collapse");
+
                 attachCollapseHandler(button);
-                pair.append(button);
+                if ($.inArray("hide-toggle", hideToggle) === -1) {
```

Comment:
Wondering if we should move this logic higher up... do we even want to create the button if `hide-toggle` is off?